src/librbd/watcher/Utils.cc

```diff
@@ -0,0 +1,14 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
Nit: doesn't look like this file is needed (i.e. no need for the explicit template instantiation below).